extensions/amp-story/0.1/test/test-utils.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * Copyright 2017 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {timeStrToMillis} from '../utils';
+
+describes.fakeWin('amp-story utils', {}, () => {
+  describe('timeStrToMillis', () => {
+    it('should return millis for a string', () => {
```

Comment:
Maybe separate these into two cases?

Also I guess we should add a test for the decimal case

Reply:
Done